testutil/ibc_testing/specific_setup.go

```diff
@@ -29,14 +30,15 @@ func ProviderAppIniter() (ibctesting.TestingApp, map[string]json.RawMessage) {
 	return testApp, appProvider.NewDefaultGenesisState(encoding.Marshaler)
 }
 
-// GaiaAppIniter implements ibctesting.AppIniter for the gaia app
-func GaiaAppIniter() (ibctesting.TestingApp, map[string]json.RawMessage) {
-	encoding := gaiaApp.MakeTestEncodingConfig()
-	app := gaiaApp.NewGaiaApp(log.NewNopLogger(), tmdb.NewMemDB(), nil, true, map[int64]bool{},
-		simapp.DefaultNodeHome, 5, encoding, simapp.EmptyAppOptions{})
-	testApp := ibctesting.TestingApp(app)
-	return testApp, gaiaApp.NewDefaultGenesisState()
-}
+// TODO: Fix Gaia version
```

Comment:
Will prob need to add this back if https://github.com/cosmos/gaia/releases/tag/v8.0.0-rc1 can pass e2e tests